src/lib/AutomateSDK.ts

```diff
@@ -283,68 +293,67 @@ export class AutomateSDK {
     return Boolean(provider?.hasOwnProperty("safe"));
   };
 
-  private async _requestAndStoreSignature() {
-    const tokenData: TokenData = {
-      message: this._signatureMessage,
-      origin: "SDK",
-    };
-    if (this.isGnosisSafeApp()) {
-      tokenData.unsignedUser = await this._signer.getAddress();
-    } else {
-      tokenData.signature = await this._signer.signMessage(
-        this._signatureMessage
-      );
-    }
-    this._token = Buffer.from(JSON.stringify(tokenData)).toString("base64");
-
-    // Set Axios headers
-    this._taskApi.defaults.headers.common[
-      "Authorization"
-    ] = `Bearer ${this._token}`;
-  }
-
-  private async _setTaskName(taskId: string, name: string): Promise<void> {
+  private async _setTaskName(
+    taskId: string,
+    name: string,
+    authToken?: string
+  ): Promise<void> {
     const path = `/tasks/${this._chainId}`;
-    await this._postTaskApi(path, { taskId, name, chainId: this._chainId });
+    await this._postTaskApi(
+      path,
+      { taskId, name, chainId: this._chainId },
+      false,
+      authToken
+    );
   }
 
-  public async renameTask(taskId: string, name: string): Promise<void> {
-    if (this.isGnosisSafeApp()) {
```

Comment:
Do we still need `isGnosisSafeApp`? 
https://github.com/gelatodigital/automate-sdk/blob/a3c25d5c09b1f468ce89deadb2ae9cea4168d4f0/src/lib/AutomateSDK.ts#L283